### DISCUSSION TITLE
Slow/failed fetch

### DISCUSSION THREAD

**Discussion by jonafeucht** [alexmercerind/dart_vlc#40] [General]
This discussion targets the slow/failed fetching.

**Issue:**
<strike>Flutter fetches vlc from the official url, which is extremly slow for me. Sometimes it works, but most times it just times out, without telling me it failed.

A potential workaround was introduced in https://github.com/alexmercerind/dart_vlc/issues/35#issuecomment-829907591. I still have to use a mirror to actually get it to successfully download.
And with every `flutter clean`, the process has to be redone.

As of now it's unclear if this is a problem I personally have or a more widespread one.
See: #35

A potential solution was introduced in PR https://github.com/alexmercerind/dart_vlc/pull/39, which uses mirrors instead of the official url. Downside, it uses mirrors.</strike>

**Alright, seems like Videolan fixed the issue with the slow download.**

The mentioned PR also addresses the problem of libvlcpp.zip not being able to be downloaded from github in China.
See: #13, #20

No idea if #39 is actually a global fix, so it would be great to get some feedback/discussion on this issue.

**Comment by alexmercerind** [comment]
So, now the things are packed within the package. No more fetching mess.
